File: Java/Assignment/Prime.java
import java.util.Scanner;

class Prime {
    public static void main(String[] args) {
        int n, f = 0,i = 2;
        Scanner read = new Scanner(System.in);
        System.out.print("Enter a number: ");
        n = read.nextInt();
        while(i<(n/i)) {
            if(n%i == 0) {
                System.out.println("The given number is not a prime number.");
                f = 1;
                break;
            }
            i++;
        }
        if(f == 0) {
            System.out.println("The given number is a prime number.");
        }
    }
}

File: python/prime.py
n = int(input("Enter a number:"))
end = n
i = 2
while i<=end:
    print(i,end)
    if n%i == 0:
        print("number is not prime")
        end = 0
        break
    end = n//i
    i+=1
if end !=0:
    print("number is prime")